linux_os/guide/services/sssd/sssd_enable_smartcards/oval/shared.xml

```diff
@@ -0,0 +1,32 @@
+<def-group oval_version="5.11">
+  <definition class="compliance" id="sssd_enable_smartcards" version="1">
+    <metadata>
+      <title>Enable Smartcards in SSSD</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
```

Comment:
the fixes use multi_platform_rhel and this uses RHEL7, which is the correct one?

Reply:
Thanks @mpreisler. Should be `RHEL7` and not `multi_platform_rhel`. I wasn't thinking here.